src/microbe_stage/Microbe.Contact.cs

```diff
@@ -383,16 +424,124 @@ public bool CanEngulf(Microbe target)
             return false;
         }
 
+        // Limit amount of things that can be engulfed at once
+        if (IngestedSizeCount > EngulfSize || IngestedSizeCount + target.EngulfSize > EngulfSize)
+            return false;
+
+        // Too many things attempted to be engulfed at once
+        if (attemptingToEngulf.Sum(e => e.EngulfSize) + target.EngulfSize > EngulfSize)
+            return false;
+
         // Disallow cannibalism
-        if (target.Species == Species)
+        if (isMicrobe && microbe!.Species == Species)
             return false;
 
         // Membranes with Cell Wall cannot engulf
         if (Membrane.Type.CellWall)
             return false;
 
         // Needs to be big enough to engulf
-        return EngulfSize >= target.EngulfSize * Constants.ENGULF_SIZE_RATIO_REQ;
+        return EngulfSize > target.EngulfSize * Constants.ENGULF_SIZE_RATIO_REQ;
+    }
+
+    public void OnAttemptedToBeEngulfed()
+    {
+        Membrane.WigglyNess = 0;
+
+        // Make the render priority of our organelles be on top of the highest possible render priority
+        // of the hostile engulfer's organelles
+        var hostile = HostileEngulfer.Value;
+        if (hostile != null)
+        {
+            foreach (var organelle in organelles!)
+            {
+                var newPriority = Mathf.Clamp(Hex.GetRenderPriority(organelle.Position) +
+                    hostile.organelles!.MaxRenderPriority, 0, Material.RenderPriorityMax);
+                organelle.UpdateRenderPriority(newPriority);
+            }
+        }
+
+        // Just in case
```

Comment:
```suggestion
        // Just in case player is engulfed again after escaping
```
Would that be a correct clarification?

Reply:
I think so.

Reply:
I assume, then you'll commit that change / do a similar change soon.